Index: kicad/kicad/prjconfig.cpp
===================================================================
--- kicad.orig/kicad/prjconfig.cpp
+++ kicad/kicad/prjconfig.cpp
@@ -89,19 +89,7 @@ void KICAD_MANAGER_FRAME::CreateNewProje
         }
         else
         {
-            // The standard path should be in the share directory for kicad. As
-            // it is normal on Windows to only have the share directory and not
-            // the kicad sub-directory we fall back to that if the directory
-            // doesn't exist
-            templatePath = wxPathOnly( wxStandardPaths::Get().GetExecutablePath() ) +
-                sep + wxT( ".." ) + sep + wxT( "share" ) + sep + wxT( "kicad" ) +
-                sep + wxT( "template" ) + sep;
-
-            if( !wxDirExists( templatePath.GetFullPath() ) )
-            {
-                templatePath = wxPathOnly( wxStandardPaths::Get().GetExecutablePath() ) +
-                sep + wxT( ".." ) + sep + wxT( "share" ) + sep + wxT( "template" ) + sep;
-            }
+            templatePath = wxPathOnly("/usr/share/kicad/template") + sep;
         }
 #else
         // Use what is provided in the bundle data dir
